### PR TITLE
Update deploy-badge.yml

### PR DESCRIPTION
**Pull request by dinho08021991**

### CODE REVIEW

**Review by Toppole6 — APPROVED** [review]

File: .github/workflows/deploy-badge.yml [hunk 1]
```diff
@@ -72,3 +72,8 @@ jobs:
         with:
           azure_static_web_apps_api_token: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN_ICY_CLIFF_0FFBDD510 }}
           action: "close"
+
+
+
+
+[![Azure Static Web Apps CI/CD](https://github.com/microsoft/app-store-badge/actions/workflows/deploy-badge.yml/badge.svg?branch=main&event=public)](https://github.com/microsoft/app-store-badge/actions/workflows/deploy-badge.yml)
```
Comment (777olivers777) [inline .github/workflows/deploy-badge.yml:79]:
```suggestion
[![Azure Static Web Apps CI/CD](https://github.com/microsoft/app-store-badge/actions/workflows/deploy-badge.yml/badge.svg?branch=main&event=public)](https://github.com/microsoft/app-store-badge/actions/workflows/deploy-badge.yml)
```**_[]()_**

**Review by 777olivers777 — APPROVED** [review]

**Review by 777olivers777 — APPROVED** [review]

**Comment by 777olivers777** [comment]
>

File: .github/workflows/deploy-badge.yml [hunk 2]
```diff
@@ -72,3 +72,8 @@ jobs:
         with:
           azure_static_web_apps_api_token: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN_ICY_CLIFF_0FFBDD510 }}
           action: "close"
```
[the remark this answers is not in the document]
Comment (wisdoms6969) [reply .github/workflows/deploy-badge.yml:74]: Hi

**Comment by wisdoms6969** [comment]
Nice

File: .github/workflows/deploy-badge.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (DNOCHA3) [reply .github/workflows/deploy-badge.yml:74]: Thanks

**Pull request closed without merging**